### ISSUE TITLE
Null Reference Exception on ElementPropertyChanged

### ISSUE THREAD

**Issue by Bharathshah** [rebeccaXam/XamForms.Controls.Calendar#101]
![IMG_0113](https://user-images.githubusercontent.com/32506518/60867758-32cb3000-a249-11e9-8b9e-ca5a76ac8f2b.PNG)

Please help me how to fix this issue?

**Comment by lubiepomaranczki** [comment]
Hey @Bharathshah 👋
is there any chance you updated Xamarin.Forms to version 4.0 or higher?

**Comment by ORobert** [comment]
Same exception here. 
Yes, a Xamarin.Forms version grater then 4.0 is installed!

**Comment by ORobert** [comment]
> Same exception here.
> Yes, a Xamarin.Forms version grater then 4.0 is installed!

Edit: The library works as expected when using a Xamarin.Forms version less then 4.0

**Comment by Bharathshah** [comment]
@lubiepomaranczki  Yes, I update to Xamarin forms 4.0 after that only I get this crash report any solution or updates for this crash?

**Comment by ORobert** [comment]
@Bharathshah Try to downgrade Xamarin.Forms to 3.6.0.539721

**Comment by lubiepomaranczki** [comment]
@ORobert, @Bharathshah Try installing [XamForms.Enhanced.Calendar ver. 1.2.1](https://www.nuget.org/packages/XamForms.Enhanced.Calendar/) which is a fork of this repository but updated. If it doesn't work, please raise an issue [here](https://github.com/lubiepomaranczki/XamForms.Controls.Calendar/issues)

Good luck! 🚀
